image-streams.yaml

```diff
@@ -0,0 +1,170 @@
+kind: List
+apiVersion: v1
+metadata:
+  name: image-streams
+  annotations:
+    description: ImageStream definition for Red Hat AMQ Streams 1.0
+    openshift.io/provider-display-name: Red Hat, Inc.
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: amqstreams10-clustercontroller-openshift
+    annotations:
+      openshift.io/display-name: AMQ Streams Cluster Controller     
+      openshift.io/provider-display-name: Red Hat, Inc.
+      version: 1.4.11
```

Comment:
what is this version here which is the same as the `xpaas` label ?

Reply:
Yes, same as the xpass label. I need to check with CE team to see if we need to follow their versioning scheme and/or what the plan is for middleware template/image stream versioning ( If we are to keep in sync with the core products or if we keep our own ). Until then, lets hold off on this PR, I might need to update a few things